src/AccessibilityInsights.SharedUx/Controls/CustomControls/CustomGridViewColumn.xaml.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Input;
+
+namespace AccessibilityInsights.SharedUx.Controls.CustomControls
+{
+    /// <summary>
+    /// A GridViewColumn styled so that "cells" look DataGridCells, complete with some custom 
+    /// resizing behavior. Meant to be used in conjunciton with <see cref="CustomGridView"/>.
```

Comment:
small - conjunction* and 'look like DataGridCells'

Reply:
```suggestion
    /// resizing behavior. Meant to be used in conjunction with <see cref="CustomGridView"/>.
```